.travis.yml

```diff
@@ -15,12 +13,16 @@ matrix:
         # Lint the first commit in the PR.
         - \[ -z "$TRAVIS_COMMIT_RANGE" \] || (echo -e '\nLinting the commit message according to the guidelines at https://goo.gl/p2fr5Q\n' && git log $TRAVIS_COMMIT_RANGE --pretty=format:'%h' --no-merges | tail -1 | xargs npx -q core-validate-commit --no-validate-metadata)
     - name: "Test Suite"
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-4.9
       install:
-        - ./configure
-        - make -j2 V=
+        - CC='ccache gcc-4.9' && CXX='ccache g++-4.9' python configure.py --verbose
+        - CC='ccache gcc-4.9' && CXX='ccache g++-4.9' make -j2 V=
```

Comment:
Maybe set these using `export CC='ccache gcc-4.9' && CXX='ccache g++-4.9'` on the first line of `install:`, and then leave these lines as they are?

Also, I’d prefer to keep `./configure` – firstly, the `.travis.yml` of a project is sometimes used by devs to figure out how to build it, because it’s usually much more to-the-point than build instructions, and secondly that way we simply test more code than without it (namely the contents of `configure`).

Reply:
I'm suspicious it's not behaving as we expect (hence it's using GCC over clang), so I'd rather has it explicit (like in Jenkins).

Ack on the `./configure`

Reply:
Testing export in `install`